kernel/Kconfig

```diff
@@ -485,6 +485,15 @@ config HEAP_MEM_POOL_SIZE
 	  dynamically allocating memory using k_malloc(). Supported values
 	  are: 256, 1024, 4096, and 16384. A size of zero means that no
 	  heap memory pool is defined.
+
+config HEAP_MEM_POOL_MIN_SIZE
+	int "The smallest blocks in the heap memory pool (in bytes)"
+	depends on HEAP_MEM_POOL_SIZE != 0
+	default 64
+	help
+	  This option specifies the size of the smallest block in the pool.
+	  Option must be the power of 2 and lower than or equal to the size
```

Comment:
remove the

Reply:
which 'the'?
s/the power/a power
?

Reply:
consulted with tech writer and fixed 
thanks

Reply:
Good :)

Reply:
:)